ddtrace/tracer/util.go

```diff
@@ -26,6 +30,8 @@ func toFloat64(value interface{}) (f float64, ok bool) {
 		return i, true
 	case int:
 		return float64(i), true
+	case int8:
+		return float64(i), true
```

Comment:
It's surprising that this was missing. We should probably go ahead and also add `uint8` for completeness.